server.js

```diff
@@ -191,6 +191,62 @@ var handlerWrapper = module.exports.handlerWrapper = function handlerWrapper(pin
       return;
     }
 
+    //This Week in Servo
+    if (message.indexOf('twis ') > -1) {
+      try {
```

Comment:
I'm not super keen on this all-encompassing try/catch. The error logs already will get a message for uncaught exceptions, so we don't really gain anything here.